tests/unit/functions/edh-marshaller.test.ts

```diff
@@ -1,7 +1,7 @@
-/* eslint-disable jest/no-conditional-expect */
 import { edhMarshaller } from '../../../src/functions/edh-marshaller';
 import { SQSService } from '../../../src/services/sqs';
 import { Context } from 'aws-lambda';
+import * as _ from '../../../src/utils/transformTechRecord';
```

Comment:
Is this used? If so why is it _?

Reply:
In order to use jest.spyOn you have to have the function be part of a module/class....
```
jest.spyOn(_, 'transformTechRecord').mockImplementation(transformRecordMock);
```